packages/api/tests/modules/user/user.service.test.ts

```diff
@@ -94,20 +96,26 @@ describe("user.service.ts", () => {
             await service.createUser("test@beta.gouv.fr");
         });
 
-        it("should reject because user email not found", async () => {
-            await expect(service.addRolesToUser("wrong@email.fr", [RoleEnum.admin])).resolves.toMatchObject({
-                success: false,
-                message: "User email does not correspond to a user",
-                code: UserServiceErrors.USER_NOT_FOUND
-            });
+        it("should throw EntityNotFoundError if user email not found", async () => {
+            const expected = new EntityNotFoundError("User Not Found");
+            let actual;
+            try {
+                actual = await service.addRolesToUser("wrong@email.fr", [RoleEnum.admin]);
+            } catch (e) {
+                actual = e;
+            }
+            expect(actual).toEqual(expected);
```

Comment:
```suggestion
            const expected = new EntityNotFoundError("User Not Found");
            const test = async () => service.addRolesToUser("wrong@email.fr", [RoleEnum.admin]);
            await expect(test)..rejects.toThrowError(expected);
```
Je trouve cette écriture + lisible

Reply:
En effet c'est plus court, j'ai essayé sans trop cherché d'utiliser toThrowError mais j'avais pas transformé le test en fonction (je l'appelai directement dans le expect et ça ne fonctionnait pas). Je modifie. ;)